system-tests/lib/cre/capabilities/evm/evm.go

```diff
@@ -59,15 +63,23 @@ func registerWithV1(_ []string, nodeSetInput *cre.CapabilitiesAwareNodeSet) ([]k
 			return nil, errors.Wrapf(selectorErr, "failed to get selector from chainID: %d", chainID)
 		}
 
+		faultyNodes := uint32((nodeSetInput.Nodes - 1) / 3)
 		capabilities = append(capabilities, keystone_changeset.DONCapabilityWithConfig{
 			Capability: kcr.CapabilitiesRegistryCapability{
 				LabelledName:   "evm" + ":ChainSelector:" + strconv.FormatUint(selector, 10),
 				Version:        "1.0.0",
-				CapabilityType: 3, // TARGET
-				ResponseType:   1, // OBSERVATION_IDENTICAL
+				CapabilityType: 0, // TRIGGER
```

Comment:
changed the `CapabilityType` from being action based to trigger, as it requires more parametrization to make it work remotely (once https://github.com/smartcontractkit/chainlink/pull/19102 is merge, both types (actions/trigger) will have to work ootb).